lib/home/profile/account/close_account.dart

```diff
@@ -130,15 +145,4 @@ class _Deleting extends ValueNotifier<_Progress> {
     await user.yeet();
     value = _Progress.done;
   }
-
-  Future<bool> checkPassword(String inputPassword) async {
-    // Replace this with the actual logic to verify the user's password
-    final storedPassword = await getUserPassword(); // Fetch the actual user password
-    return inputPassword == storedPassword;
-  }
-
-  Future<String> getUserPassword() async {
-    // Fetch the user password from your authentication service
-    return 'user_actual_password';
-  }
```

Comment:
It's not possible to fetch the password from the database—if you could do that, it'd make hacking a lot easier!

Instead, a "hashing algorithm" is used to (1) ensure that passwords aren't stored as plain text and (2) make submitting a password a more computationally intensive process, so people are less motivated to try to crack them.

Luckily, Firebase Authentication takes care of that for us, so I was able to implement the logic to verify the user's password!